dask/dataframe/core.py

```diff
@@ -739,6 +739,11 @@ def repartition(self, divisions=None, npartitions=None, force=False):
             raise ValueError(
                 "Provide either divisions= or npartitions= to repartition")
 
+    @derived_from(pd.DataFrame)
+    def select_dtypes(self, include=None, exclude=None):
```

Comment:
Since this method only works on `pd.DataFrame`, it should be a method on the `DataFrame` class instead of the `_Frame` superclass.